folium/plugins/realtime.py

```diff
@@ -98,11 +104,13 @@ def __init__(
         get_feature_id: Optional[JsCode] = None,
         update_feature: Optional[JsCode] = None,
         remove_missing: bool = False,
+        container: Optional[Layer] = None,
```

Comment:
I have a sleight concern about this allowing any `Layer` object. In the Leaflet plugin documentation I read the container argument should be a (sub)class of L.LayerGroup. That's more strict than Layer.

I'm now thinking I'd rather limit this here to the exact classes that are allowed. I suppose those are not that many, so we can list them here and in the docs/docstring. And possibly add a check that raises an error on incorrect types. That way we avoid user questions about what is and isn't possible here, and save ourselves effort answering issues in the future. Looking at which classes that are, I'm thinking probably just `FeatureGroup`, `GeoJson`, `MarkerCluster`?

Reply:
Good point about the `L.LayerGroup`. I changed it accordingly. 

I'd prefer not limit the container to only a limited set of classes. This would limit users of the package if they want to define their own containers (or extend an existing container). The python interface for the container is minimal. We only use `get_name` and  `add_to`. I updated the documentation to reflect the requirements on the JavaScript side.

Reply:
Alright, that's fair. As long as it's well documented, we should be good.

I don't see your changes by the way, maybe you still need to push them?

Reply:
I messed up, I accidentally pushed something that did not work. There is no LayerGroup in folium.